theseus/geometry/functional/se3.py

```diff
@@ -483,6 +483,60 @@ def backward(cls, ctx, grad_output):
 vee = lie_group.UnaryOperatorFactory(_module, "vee")
 
 
+# -----------------------------------------------------------------------------
+# Compose
```

Comment:
Minor point but maybe once all these PRs are merged we should sort all these blocks (maybe alphabetically by function name?) so that they appear in a consistent order between, for example, so3.py and se3.py.